lib/fizzy/value.hpp

```diff
@@ -57,18 +57,6 @@ constexpr uint32_t Value::as<uint32_t>() const noexcept
     return i32;
 }
 
-template <>
-constexpr unsigned long Value::as<unsigned long>() const noexcept
```

Comment:
Added these as "quick fix" to see which ones the compiler couldn't correctly match :)